webapp/tests/utils/channel_utils.test.jsx

```diff
@@ -0,0 +1,404 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import * as Utils from 'utils/channel_utils.jsx';
+import TeamStore from 'stores/team_store.jsx';
+import UserStore from 'stores/user_store.jsx';
+import ChannelStore from 'stores/channel_store.jsx';
+import Constants from 'utils/constants.jsx';
+
+describe('Channel Utils', () => {
+    describe('showDeleteOption', () => {
+        test('all users can delete channels on unlicensed instances', () => {
+            global.window.mm_license = {IsLicensed: 'false'};
```

Comment:
Is there any risk in changing global state like this for the purposes of a test? Could I be accidentally poisoning the global state for some other test by doing this?